### ISSUE TITLE
Implement GetSchemaTable

### ISSUE THREAD

**Issue by Topcoder01** [mysql-net/MySqlConnector#307]
Dim oFilesToProcessDataTable As New DataTable
 Dim oMySqlCommand As New MySqlCommand
 oFilesToProcessDataTable.Load(oMySqlCommand.ExecuteReader)

The above tosses an "command not implemented" or something very similar to that, I forgot :( 

If this isn't supported is their another easy way to populate a DataTable with this connector?

**Comment by bgrainger** [comment]
The stack trace is:

```
NotSupportedException: Specified method is not supported.
   at MySql.Data.MySqlClient.MySqlDataReader.GetSchemaTable()
   at System.Data.ProviderBase.SchemaMapping..ctor(DataAdapter adapter, DataSet dataset, DataTable datatable, DataReaderContainer dataReader, Boolean keyInfo, SchemaType schemaType, String sourceTableName, Boolean gettingData, DataColumn parentChapterColumn, Object parentChapterValue)
   at System.Data.Common.DataAdapter.FillMappingInternal(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 schemaCount, DataColumn parentChapterColumn, Object parentChapterValue)
   at System.Data.Common.DataAdapter.FillMapping(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 schemaCount, DataColumn parentChapterColumn, Object parentChapterValue)
   at System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   at System.Data.Common.DataAdapter.Fill(DataTable[] dataTables, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   at System.Data.DataTable.Load(IDataReader reader, LoadOption loadOption, FillErrorEventHandler errorHandler)
```

This may actually be part of #183; if not part of that work, it's closely related.

**Comment by bgrainger** [comment]
Fixed in [0.27.0](https://github.com/mysql-net/MySqlConnector/releases/tag/0.27.0).
